src/raygun.utilities/errorUtilities.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * @prettier
+ */
+
+window.raygunErrorUtilitiesFactory = function (window, Raygun) {
```

Comment:
Would be good to handle the case where what about the case where this is run in an non-browser environment?